contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,27 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
```

Comment:
@zacps Hmm, maybe the username mention part is unclear here - doesn't the bot react to `@github-detail`?